app/services/api/concerns/teachers/shared_action.rb

```diff
@@ -24,6 +24,17 @@ module SharedAction
       }, allow_blank: true
     end
 
+    def training_period
```

Comment:
Can we revert this back to `private` now that we're not using this concern for declarations